distribution/tools/server-cli/src/test/java/org/elasticsearch/server/cli/APMJvmOptionsTests.java

```diff
@@ -108,7 +108,6 @@ public void testExtractSecureSettings() {
 
     public void testExtractSettings() throws UserException {
         Function<String, Settings.Builder> buildSettings = (prefix) -> Settings.builder()
-            .put("tracing.apm.enabled", true)
             .put(prefix + "server_url", "https://myurl:443")
```

Comment:
Why is this no longer necessary?

Reply:
Just removed it because it's doesn't seem relevant for the test case